src/patternfly/upgrade-examples/PopoverUpgradeExamples/examples/popover-upgrade-examples-example2.hbs

```diff
@@ -0,0 +1,21 @@
+{{#> popover-upgrade-examples}}
+<div class="pf-c-popover pf-m-right" role="dialog" aria-modal="true" aria-labelledby="popover-right-header" aria-describedby="popover-right-body">
+  <div class="pf-c-popover__arrow">
+  </div>
+  <div class="pf-c-popover__content">
+    <div class="pf-c-popover__close">
+      <button class="pf-c-button pf-m-plain" aria-label="Close">
+        <i class="fas fa-times"></i>
+      </button>
+    </div>
+    <header class="pf-c-popover__header">
+      <h1 class="pf-c-popover__header-title" id="popover-right-header">
+        Popover Header
+      </h1>
+    </header>
+    <div class="pf-c-popover__body" id="popover-right-body">
+      Popover Body <br> <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis.
```

Comment:
Can you change the body of the examples for both to be:  (remove Popover Body)
"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis." like in the pf4 component.

Reply:
- "Popover Body" text removed.
- Lorem ipsum content has been added.